pandas/tests/frame/methods/test_explode.py

```diff
@@ -260,18 +272,32 @@ def test_explode_sets():
             },
             list("aaabcdde"),
         ),
+        (
+            {
+                "A": [[0, 1, 2], [], np.nan, [], (3, 4), np.nan, []],
+                "B": 1,
+                "C": [["a", "b", "c"], 2, "foo", [], ["d", "e"], np.nan, np.nan],
+            },
+            list("abcdefg"),
+            list("AC"),
+            {
+                "A": pd.Series(
+                    [0, 1, 2, np.nan, np.nan, np.nan, 3, 4, np.nan, np.nan],
+                    index=list("aaabcdeefg"),
+                    dtype=object,
+                ),
+                "B": 1,
+                "C": ["a", "b", "c", 2, "foo", np.nan, "d", "e", np.nan, np.nan],
+            },
+            list("aaabcdeefg"),
+        ),
     ],
 )
-def test_multi_columns(input_subset, expected_dict, expected_index):
+def test_multi_columns(
+    input_dict, input_index, input_subset, expected_dict, expected_index
+):
     # GH 39240
```

Comment:
In regards to https://github.com/pandas-dev/pandas/pull/46791/files#r851832909, add the issue number here; i.e. `# GH 39240, 46084`